samples/nrf9160/aws_fota/prj.conf

```diff
@@ -0,0 +1,50 @@
+# General config
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_MPU_STACK_GUARD=y
+
+CONFIG_IMG_MANAGER=y
+CONFIG_BOOTLOADER_MCUBOOT=y
+
+# Enable DOWNLOAD libs
+CONFIG_NRF_DOWNLOAD_CLIENT=y
+CONFIG_NRF_DOWNLOAD_MAX_RESPONSE_SIZE=2048
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
```

Comment:
You don't need this with offloaded sockets, please remove.

Reply:
removed.